sys/net/ble/skald/Kconfig

```
@@ -0,0 +1,31 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_SKALD
+    bool "Configure SKALD"
+    depends on MODULE_SKALD
+    help
+        Configure Skald, BLE advertising stack, using Kconfig.
+
+if KCONFIG_MODULE_SKALD
+
+config SKALD_INTERVAL
+    int "Advertising interval in microseconds"
+    default 1000000
+    help
+        Configure advertising interval in microseconds. Default value is 1
+        second which is 1000000 microseconds.
+
+config ADV_CH_37_DISABLE
+    bool "Disable channel 37"
```

Here and bellow, it seems worth clarifying:
```suggestion
    bool "Disable advertising on channel 37"
```

---

Channels 37, 38 and 39, as you are aware is used for advertising. Should we assume that a user is aware of the same ?

---

I don't understand how that assumption changes the fact of being more specific here or not.

---

I thought it was reassuring. If you say so I shall change the same

---

Makes sense. Seems like I got myself confused :bowtie: